content/300-guides/050-database/300-seed-database.mdx

```diff
@@ -165,14 +130,14 @@ To seed your database with TypeScript:
 
 <a id="compiler-options"></a>
 
-4. Optionally, add additional `--compiler-options` to the script that Prisma uses to run the seed script ([2.18.0](https://github.com/prisma/prisma/releases/tag/2.18.0) and later). This makes `prisma db seed` work with tools that have specific requirements when used with TypeScript, such as Next.js. The name of the script is `ts-node`:
+4. Add the `prisma.seed` to your `package.json` file:
```

Comment:
```suggestion
4. Add the `prisma.seed` key to your `package.json` file:
```
Is `key` the right name for this? Or `property`?

Reply:
Perhaps property. prisma is the key on that property. But its also another object which has seed as a key 😁

Reply:
Maybe field is better actually